tutorials/terminology.md

```diff
@@ -63,10 +63,10 @@ to developers touching the source code.
     the world, while the **primary** runner is keeping all secondaries in sync.
     Worlds that are not split across runners don't have a primary runner.
 
-* **[Event manager](classignition_1_1gazebo_1_1EventManager.html)**:
+* \ref gz::sim::EventManager "Event manager":
     Manages events that can be sent across systems and the server. Plugins can
     create and emit custom
-    [Event](https://gazebosim.org/api/common/3.0/classignition_1_1common_1_1Event.html)s
+    \ref gz::common::Event "Event"s
```

Comment:
Should this be "Events" with the s included?

Reply:
I kept it the way it was. I think they wanted to make the link text `Event` for the class name, so as to not make people think the class name is `Events`.